### DISCUSSION TITLE
Share with friends

### DISCUSSION THREAD

**Discussion by ohnegesetz** [IBAX-io/go-ibax#23] [Rewards]
I invited about 5 friends of mine. What should my friends do for me to earn more points? Up together!
my wallet adress : 0388-2965-0130-5408-2967
my friens wallet adress : 0046-0564-9037-8242-9328

**Comment by HalloKyle** [comment]
> I invited about 5 friends of mine. What should my friends do for me to earn more points? Up together! my wallet adress : 0388-2965-0130-5408-2967 my friens wallet adress : 0046-0564-9037-8242-9328

Your friends can join IBAX airdrop campaign and get IBAX points or invite more friends to get IBXC coins

**Comment by Emirxt** [comment]
Arkadaşlarınız Airdrop görevlerini yapsın ve daha fazla insan davet etsin bu büyük projeye

**Comment by hemresenol** [comment]
try harder for points

**Comment by Emirxt** [comment]
👍👍

23 Oca 2022 Paz ÖS 2:33 tarihinde hemresenol ***@***.***>
şunu yazdı:

**Comment by cihanaslan0** [comment]
Paylaşalım aga
